src/markdown-pages/explore-docs/nerdpack-file-structure.mdx

```diff
@@ -144,7 +144,7 @@ The launcher icon that appears on the **Apps** page in New Relic One when an app
 
 Launchers have their own file structure. Note that:
 
-- A launcher is not required; as an alternative to using a launcher, you can [associate your application with a monitored entity](https://docs.newrelic.com/docs/new-relic-one/use-new-relic-one/build-new-relic-one/new-relic-one-application-nerdpack-file-structure#nerdlet-json).
+- A launcher is not required; as an alternative to using a launcher, you can [associate your application with a monitored entity](#nr1json).
```

Comment:
Could we link them [here](/build-apps/attach-nerdlet-to-entities/). I think that's a more useful guide.

Reply:
Sorry, [here](https://developer.newrelic.com/build-apps/attach-nerdlet-to-entities/) 😄

Reply:
ok done, fire at will